Source/Cherry.spoon/init.lua

```diff
@@ -0,0 +1,158 @@
+--- === Cherry ===
+---
+--- Cherry tomato (a tiny Pomodoro) -- a Pomodoro Timer for the menubar
+---
+--- Download: [https://github.com/Hammerspoon/Spoons/raw/master/Spoons/Cherry.spoon.zip](https://github.com/Hammerspoon/Spoons/raw/master/Spoons/Cherry.spoon.zip)
+---
+
+local obj = {}
+obj.__index = obj
+
+-- Metadata
+obj.name = "Cherry"
+obj.version = "0.1"
+obj.author = "Daniel Marques <danielbmarques@gmail.com>"
```

Comment:
```suggestion
obj.author = "Daniel Marques <danielbmarques@gmail.com> and Omar El-Domeiri <omar@doesnotexist.com>"
```

Great job, thanks! 🍻